API/app/Http/Requests/RetrieveForms.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RetrieveForms extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
tipo di ritorno `booleano`

Noto che questa funzione e' solo uno scheletro dell'autenticazione, chissa' se in futuro l'OPIS-Manager avra' un pannello utente per visualizzare le proprie materie ed i propri corsi e prof.
E magari l'utente potra anche votare materie e prof.
Ma ci vorrebbe troppo lavoro per fare un progetto simile xD

Reply:
Meglio mantenerlo snello...